Source/carthage/Build.swift

```diff
@@ -224,24 +224,44 @@ public enum BuildPlatform {
 	/// Build only for watchOS.
 	case watchOS
 
-	/// The `Platform` corresponding to this setting.
-	public var platform: Platform? {
+	case Multiple([BuildPlatform])
```

Comment:
> It seems that `ArgumentType` cannot handle array type, so we need a wrapper type

Ah, okay! That seems like a good reason to use your original implementation. Sorry!